sdk/src/test/kotlin/io/provenance/scope/sdk/AffiliateRepositoryTest.kt

```diff
@@ -10,7 +10,7 @@ import io.provenance.scope.sdk.AffiliateRepository
 import java.security.PublicKey
 
 class AffiliateRepositoryTest : WordSpec() {
-    lateinit var affiliateRepository: AffiliateRepository
+    lateinit var affiliateRepository: io.provenance.scope.sdk.AffiliateRepository
```

Comment:
for some reason the package statement for this file seems weird (might have been my bad), seems like it should just be `io.provenance.scope.sdk` given the directory structure

Reply:
Will check that quickly!